edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubernetesExperimentalCreatePodParameters.cs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.Azure.Devices.Edge.Agent.Kubernetes.EdgeDeployment;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json.Linq;
+
+    public class KubernetesExperimentalCreatePodParameters
+    {
+        public Option<IDictionary<string, string>> NodeSelector { get; }
+
+        public KubernetesExperimentalCreatePodParameters(Option<IDictionary<string, string>> nodeSelector)
+        {
+            this.NodeSelector = nodeSelector;
+        }
+
+        public static Option<KubernetesExperimentalCreatePodParameters> Parse(IDictionary<string, JToken> other)
+            => Option.Maybe(other).FlatMap(options => options.Get("k8s-experimental").FlatMap(ParseParameters));
```

Comment:
nit pick... Define the k8s-experimental const string and refer here.

Reply:
Sure